src/app/components/app-settings/app-settings.component.ts

```diff
@@ -37,10 +37,20 @@ export class AppSettingsComponent implements OnInit {
       name: null,
       email: null
     };
+
+    this.auth.loginStatus.subscribe(loggedIn => {
```

Comment:
hmm I don't think so. In case you log out and login with other user and then go to settings again in your case there will still be properties from previous user... That's why you have to refresh object in case of new login...